src/app/modules/form/admin-form/admin-form.controller.ts

```diff
@@ -2465,3 +2466,47 @@ export const handleUpdateStartPage = [
   }),
   _handleUpdateStartPage,
 ] as ControllerHandler[]
+
+/**
+ * Handler to retrieve the free sms counts used by a form's administrator
+ * This is the controller for GET /admin/forms/:formId/verified-sms/count/free
+ * @param formId The id of the form to retrieve the free sms counts for
+ * @returns 200 with free sms counts when successful
+ * @returns 404 when the formId is not found in the database
+ * @returns 500 when a database error occurs during retrieval
+ */
+export const handleGetFreeSmsCountForFormAdmin: ControllerHandler<
+  {
+    formId: string
+  },
+  ErrorDto | number
+> = (req, res) => {
+  const { formId } = req.params
+  const logMeta = {
+    action: 'handleGetFreeSmsCountForFormAdmin',
+    ...createReqMeta(req),
+    formId,
+  }
+
+  // Step 1: Check that the form exists
+  return (
+    FormService.retrieveFormById(formId)
+      // Step 2: Retrieve the free sms count
+      .andThen(({ admin }) => {
+        return SmsService.retrieveFreeSmsCounts(String(admin))
+      })
+      // Step 3: Map/MapErr accordingly
+      .map((freeSmsCountForAdmin) =>
+        res.status(StatusCodes.OK).json(freeSmsCountForAdmin),
```

Comment:
Changes seem to be reverted here as compared to #2500

Reply:
yeap, #2500 got merged into `feat/sms-limiting` cos i didn't want to tack on more stuff here.

Reply:
cherry picked so #2500 is here; added a missing dependency in